crates/biome_html_parser/tests/html_specs/error/frontmatter_bogus.html.snap

```diff
@@ -0,0 +1,168 @@
+                        name: HtmlTagName {
+                            value_token: HTML_LITERAL@55..59 "body" [] [],
+                        },
+                        r_angle_token: R_ANGLE@59..60 ">" [] [],
+                    },
+                },
+            ],
+            closing_element: HtmlClosingElement {
+                l_angle_token: L_ANGLE@60..62 "<" [Newline("\n")] [],
+                slash_token: SLASH@62..63 "/" [] [],
+                name: HtmlTagName {
+                    value_token: HTML_LITERAL@63..67 "html" [] [],
+                },
+                r_angle_token: R_ANGLE@67..68 ">" [] [],
+            },
+        },
+    ],
+    eof_token: EOF@68..69 "" [Newline("\n")] [],
+}
+```
+
+## CST
+
+```
+0: HTML_ROOT@0..69
+  0: (empty)
+  1: HTML_BOGUS_FRONTMATTER@0..23
+    0: FENCE@0..3 "---" [] []
+    1: HTML_LITERAL@3..20 "const b = \"bar\"\n" [Newline("\n")] []
+    2: FENCE@20..23 "---" [] []
+  2: (empty)
+  3: HTML_ELEMENT_LIST@23..68
+    0: HTML_ELEMENT@23..68
+      0: HTML_OPENING_ELEMENT@23..30
+        0: L_ANGLE@23..25 "<" [Newline("\n")] []
+        1: HTML_TAG_NAME@25..29
+          0: HTML_LITERAL@25..29 "html" [] []
+        2: HTML_ATTRIBUTE_LIST@29..29
+        3: R_ANGLE@29..30 ">" [] []
+      1: HTML_ELEMENT_LIST@30..60
+        0: HTML_ELEMENT@30..45
+          0: HTML_OPENING_ELEMENT@30..38
+            0: L_ANGLE@30..33 "<" [Newline("\n"), Whitespace("\t")] []
+            1: HTML_TAG_NAME@33..37
+              0: HTML_LITERAL@33..37 "head" [] []
+            2: HTML_ATTRIBUTE_LIST@37..37
+            3: R_ANGLE@37..38 ">" [] []
+          1: HTML_ELEMENT_LIST@38..38
+          2: HTML_CLOSING_ELEMENT@38..45
+            0: L_ANGLE@38..39 "<" [] []
+            1: SLASH@39..40 "/" [] []
+            2: HTML_TAG_NAME@40..44
+              0: HTML_LITERAL@40..44 "head" [] []
+            3: R_ANGLE@44..45 ">" [] []
+        1: HTML_ELEMENT@45..60
+          0: HTML_OPENING_ELEMENT@45..53
+            0: L_ANGLE@45..48 "<" [Newline("\n"), Whitespace("\t")] []
+            1: HTML_TAG_NAME@48..52
+              0: HTML_LITERAL@48..52 "body" [] []
+            2: HTML_ATTRIBUTE_LIST@52..52
+            3: R_ANGLE@52..53 ">" [] []
+          1: HTML_ELEMENT_LIST@53..53
+          2: HTML_CLOSING_ELEMENT@53..60
+            0: L_ANGLE@53..54 "<" [] []
+            1: SLASH@54..55 "/" [] []
+            2: HTML_TAG_NAME@55..59
+              0: HTML_LITERAL@55..59 "body" [] []
+            3: R_ANGLE@59..60 ">" [] []
+      2: HTML_CLOSING_ELEMENT@60..68
+        0: L_ANGLE@60..62 "<" [Newline("\n")] []
+        1: SLASH@62..63 "/" [] []
+        2: HTML_TAG_NAME@63..67
+          0: HTML_LITERAL@63..67 "html" [] []
+        3: R_ANGLE@67..68 ">" [] []
+  4: EOF@68..69 "" [Newline("\n")] []
+
+```
+
+## Diagnostics
+
+```
+frontmatter_bogus.html:1:1 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × invalid syntax
```

Comment:
This message could also be improved. Something like "Unexpected frontmatter in normal HTML"